.github/workflows/build-sw-container.yml

```diff
@@ -0,0 +1,49 @@
+name: Build and Push Docker image
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
```suggestion
```

Looking at the documentation, it looks like branch and tag restriction lists are or'ed, not and'ed together https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestbranchestags